pkg/arvo/app/dm-hook.hoon

```diff
@@ -0,0 +1,123 @@
+::  invite-hook [landscape]: receive invites from any source
+::
+::    only handles %invite actions:
+::    - can be poked by the host team to send an invite out to someone.
+::    - can be poked by foreign ships to send an invite to us.
+::
+/+  default-agent, dbug, store=graph-store, graphlib=graph, agentio, resource
+::
+|%
++$  state-0  [%0 ~]
++$  card  card:agent:gall
+--
+::
+=|  state-0
+=*  state  -
+%-  agent:dbug
+^-  agent:gall
+::
+|_  =bowl:gall
++*  this  .
+    def   ~(. (default-agent this %|) bowl)
+    gra   ~(. graphlib bowl)
+    io    ~(. agentio bowl)
+    pass  pass:io
+::
+++  on-init  
+  :_  this
+  :_  ~
+  %+  poke-our:pass  %graph-store
+  %+  update:cg:gra  now.bowl
+  :+  %add-graph  [our.bowl %inbox]
```

Comment:
Let's change the name of this from `[our.bowl %inbox]` to `[our.bowl %dm-inbox]`

Reply:
Bump